hockeyapp/app.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, token, app_id=None):
         if app_id:
             self._check_app_id(app_id)
         self._app_id = app_id
-        super(Application, self).__init__(token)
+        super(self.__class__, self).__init__(token)
```

Comment:
iirc this is for compatibility between python versions. we did update print statements, though, so i'm not sure how much the repo maintainer is trying to keep things working for both 2 and 3